### DISCUSSION TITLE
When updating Foobar to new version, how to transfer the theme??

### DISCUSSION THREAD

**Discussion by TeddyRayBullard** [TT-ReBORN/Georgia-ReBORN#118] [Q&A] [answered]
Learned the hard way that foobar updates break the theme.  Forgive me if you have answered this, ive looked for the answer everywhere.  thank you!!

**Comment by TT-ReBORN** [comment, marked as answer]
It's quite easy, you just install the new foobar version to a new temp folder and copy all of it's content to your existing one.

For novice users: 
Before you update, I always recommend to make backups from your existing foobar directories:
profile\configuration
profile\library-v2.0
profile\playlists-v2.0

If the theme won't startup:
Paste again the entire profile folder from the zip file, and copy your backups in your profile folder again.
Start foobar and all should be good.

-TT
